packages/assistant-stream/src/core/serialization/data-stream/DataStream.ts

```
@@ -0,0 +1,347 @@
+              break;
+            }
+
+            case DataStreamStreamChunkType.ToolCallResult: {
+              const { toolCallId, result } = value as {
+                toolCallId: string;
+                result: any;
+              };
+              const toolCallController = toolCallControllers.get(toolCallId);
+              if (!toolCallController)
+                throw new Error(
+                  "Encountered tool call result with unknown id: " + toolCallId,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)